scripts/vaccine-upload/vaccine-upload.py

```diff
@@ -0,0 +1,36 @@
+from urllib.parse import urljoin
+
+import requests
+
+MSISDN_FILENAME = "users.csv"
+TURN_URL = "https://whatsapp.turn.io/"
+TURN_TOKEN = (
```

Comment:
Not sure if you wanted to commit this?

Reply:
Yeah thanks, was a mistake. Let me remove it and revoke the token